package.json

```diff
@@ -81,6 +81,7 @@
     "react-dom": "^16.2.0",
     "react-router-dom": "^4.0.0",
     "react-test-renderer": "^16.2.0",
+    "replace-in-file": "^3.4.0",
```

Comment:
Could this be removed?

Reply:
Ah yes, need to remove that file, I use it for doing mass changes to the changelog, but I forgot it was still in the package.json file.

Reply:
Removed in this commit: https://github.com/cerner/terra-core/pull/1717/commits/d70e6d2f491dfc308024101b48040fae3db3834b